test/built-ins/NativeErrors/toString.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 Ecma International. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: NativeError prototypes do not have an own toString method.
```

Comment:
missing esid

Reply:
Which esid should I put?

Reply:
Do I put the esid of Error.prototype.toString?